src/client/src/css/jobListing.css

```diff
@@ -0,0 +1,89 @@
+
+.Container{
+
+    display: flex;
```

Comment:
While the page is responsive, it looks messy on a smaller device. Open Inspect on Chrome/Firefox to check this, and toggle Device Preview.

![image](https://user-images.githubusercontent.com/78051064/219803612-cfc31c4b-8d91-4800-86bb-2f1ff8b80632.png)

Reply:
Made the pages responsive for smaller devices.
![Screen Shot 2023-02-17 at 7 06 40 PM](https://user-images.githubusercontent.com/14958097/219819252-1863aea3-03e4-4323-b48e-8471aedaee68.png)
For job listing and job application